apps/web/vibes/soul/form/swatch-radio-group/index.tsx

```diff
@@ -33,12 +33,11 @@ export const SwatchRadioGroup = React.forwardRef<
 >(({ id, label, options, errors, className, ...rest }, ref) => {
   return (
     <div className={clsx('space-y-2', className)}>
-      {label !== undefined && label !== '' && <Label htmlFor={id}>{label}</Label>}
+      {label !== undefined && label !== '' && <Label>{label}</Label>}
```

Comment:
We want to keep the `htmlFor` for accessibility. We could use `useId` instead of removing the prop

Reply:
Fixed.